spec/api_client.js

```diff
@@ -101,7 +101,7 @@ describe('api client', function () {
 
   it('should direct post requests through the proxy when set', function (done) {
     var urlBase = 'https://api.notifications.service.gov.uk',
-      proxyConfig = { host: 'addressofmyproxy.test'},
+      proxyConfig = { host: 'addressofmyproxy.test', protocol: 'http:'},
```

Comment:
Massively pedantic but can we change this to match the format in the axios README (without the colon)?
```suggestion
      proxyConfig = { host: 'addressofmyproxy.test', protocol: 'http'},
```